ext/rbs_extension/parser.c

```diff
@@ -1780,25 +1785,23 @@ VALUE parse_mixin_member(parserstate *state, bool from_interface, position comme
   range keyword_range;
   range args_range = NULL_RANGE;
   bool reset_typevar_scope;
+  enum TokenType type;
 
   member_range.start = state->current_token.range.start;
   comment_pos = nonnull_pos_or(comment_pos, member_range.start);
 
+  type = state->current_token.type;
```

Comment:
Can we drop the forward declaration?

```suggestion
  enum TokenType type = state->current_token.type;
```
```suggestion
  member_range.start = state->current_token.range.start;
  comment_pos = nonnull_pos_or(comment_pos, member_range.start);

  enum TokenType type = state->current_token.type;
```

Reply:
It's the style of the original code, I'd prefer to follow it for now to avoid inconsistencies. We can clean all of it later 👍